api.planx.uk/session.ts

```diff
@@ -0,0 +1,12 @@
+import { RequestHandler } from "http-proxy-middleware";
+
+// this middleware registers dummy methods on req.session to resolve passport/cookie-session incompatibility
+export const registerSessionStubs: RequestHandler = (req, _res, next): void => {
+  if (req.session && !req.session.regenerate) {
+    req.session.regenerate = (cb: (err?: any) => void) => cb();
+  }
+  if (req.session && !req.session.save) {
+    req.session.save = (cb: (err?: any) => void) => cb();
```

Comment:
nit: I think these `any` might be able to use built-in `Error` interface? (Haven't tested locally, `any` isn't a blocker to merging if not!)

Reply:
aye this is a nice little improvement, will get it in!